ui/src/components/Equipment/SampleChanger.js

```diff
@@ -1,313 +1,270 @@
-/* eslint-disable react/jsx-handler-names */
-import React from 'react';
+import React, { useState } from 'react';
 import { Card, Dropdown, Form, Button, DropdownButton } from 'react-bootstrap';
 
 import { Menu, Item, Separator, contextMenu } from 'react-contexify';
 
-import './Equipment.css';
-
-export class SampleChangerTree extends React.Component {
-  render() {
-    let titleBackground;
-
-    switch (this.props.state) {
-      case 'READY': {
-        titleBackground = 'success';
-
-        break;
-      }
-      case 'MOVING': {
-        titleBackground = 'warning';
-
-        break;
-      }
-      case 'DISABLED': {
-        titleBackground = 'default';
-
-        break;
-      }
-      default: {
-        titleBackground = 'danger';
-      }
-    }
-
-    return (
-      <Card
-        className="mb-3"
-        style={{ marginTop: '0.5em' }}
-        bg={titleBackground}
-      >
-        <Card.Header>{this.props.title}</Card.Header>
-        {this.props.children}
-      </Card>
-    );
+import styles from './equipment.module.css';
+
+// function SampleChangerTree(props) {
+//   let titleBackground;
+
+//   switch (props.state) {
+//     case 'READY': {
+//       titleBackground = 'success';
+
+//       break;
+//     }
+//     case 'MOVING': {
+//       titleBackground = 'warning';
+
+//       break;
+//     }
+//     case 'DISABLED': {
+//       titleBackground = 'default';
+
+//       break;
+//     }
+//     default: {
+//       titleBackground = 'danger';
+//     }
+//   }
+
+//   return (
+//     <Card className="mb-3" style={{ marginTop: '0.5em' }} bg={titleBackground}>
+//       <Card.Header>{props.title}</Card.Header>
+//       {props.children}
+//     </Card>
+//   );
+// }
```

Comment:
Why?

Reply:
This is unused and i was not sure why.
was not my code so Did not want to take the responsible to remove at first place .
I wanted to asked first what was the intention of it usage.  then i forgot